dripline/extensions/thermo_fisher_endpoint.py

```diff
@@ -0,0 +1,82 @@
+from dripline.core import Entity, calibrate, ThrowReply
+
+import logging
+logger = logging.getLogger(__name__)
+
+__all__ = []
+__all__.append("ThermoFisherGetHexEntity")
+__all__.append("ThermoFisherGetEntity")
+
+
+class ThermoFisherGetHexEntity(Entity):
```

Comment:
possibly more clear to change order to `ThermoFisherHexGetEntity` (since in other implementations there's never been another modifier between "Get" and "Entity")

Reply:
switched the Hex and Get